aerogear/mobile-developer-console
# ✨ implement the logout function

## Motivation

https://issues.jboss.org/browse/AEROGEAR-7993

## What

Implement the logout function

## Why

need to allow users to logout

## How

Redirect user to the "/oauth/sign_in" page. It will clear the session cookie and return the login view.

## Verification Steps

1. Login to https://mdc-server-psturc-test.community.skunkhenry.com
2. Perform logout after logged in.
3. Check the session cookie is gone.

## Checklist:
